src/pages/trade/ui/order-form/order-input.tsx

```diff
@@ -0,0 +1,99 @@
+import { forwardRef, useId } from 'react';
+import { useComponentSize } from 'react-use-size';
+import SpinnerIcon from '@/shared/assets/spinner-icon.svg';
```

Comment:
We've been generally using this for a spinner fyi:

```ts
import { LineWave } from 'react-loader-spinner';
```

Reply:
or does that spinner come from the new UI library?

Reply:
I just tried the LineWave, it's not a good fit here and yes this svg is from figma.